### DISCUSSION TITLE
How to display tags inside the input box?

### DISCUSSION THREAD

**Discussion by loongmxbt** [directus/directus#12359] [Archive]
In the tags component (field), when I enter some tag, it appears below the input box like this.
<img width="846" alt="image" src="https://user-images.githubusercontent.com/5986600/159939350-00e94d85-41c8-4306-a83c-0b4de77c5ec0.png">
I think display tags in the input box is very common in many ui libraries. Is there a way to config it? Or should I build a custom replacement for tags field to achieve this?
<img width="462" alt="image" src="https://user-images.githubusercontent.com/5986600/159939144-497b0c44-de9b-436c-a532-8c2d7c03767e.png">

**Comment by rijkvanzanten** [comment]
There's currently no way to configure that in the interface. Creating a custom interface is the only way 👍🏻
